packages/material-ui/src/ButtonBase/ButtonBase.test.js

```diff
@@ -757,25 +756,6 @@ describe('<ButtonBase />', () => {
     });
   });
 
-  describe('rerender', () => {
-    beforeEach(() => {
-      rerender.spy();
-    });
-
-    afterEach(() => {
-      rerender.reset();
-    });
-
-    it('should not rerender the TouchRipple', () => {
-      const wrapper = render(<ButtonBase>foo</ButtonBase>);
-      wrapper.setProps({
-        children: 'bar',
-      });
-
-      expect(rerender.updates.filter(update => update.displayName !== 'NoSsr')).to.have.lengthOf(1);
```

Comment:
This didn't work anymore since we don't use class components anymore. Using the profiler in tests will serve better in monitoring performance.